TestRailAutomationTest/Driver/DriverFactory.cs

```diff
@@ -0,0 +1,28 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+using OpenQA.Selenium.Firefox;
+using TestRailAutomationTest.Service;
+
+
+namespace TestRailAutomationTest.Driver;
+
+public class DriverFactory
+{
+    private static IWebDriver? _driver;
```

Comment:
I think we should name it like `driver`
By the way static driver will be working only in one thread 
If we want use test parallelization we should have driver instance for each test